app.rb

```diff
@@ -0,0 +1,49 @@
+require "bundler/setup" # require all the gems we'll be using for this app from the Gemfile. Obviates the need for `bundle exec`
+require "pg" # postgres db library
+require "active_record" # the ORM
+require "pry" # for debugging
+require 'sinatra'
+require 'sinatra/reloader'
+
+require_relative "db/connection.rb" # require the db connection file that connects us to PSQL, prior to loading models
+require_relative "models/apartment"
+require_relative "models/tenant"
+
+get '/' do
+ erb :index
+end
+
+get '/apartments' do
+  @apartments = Apartment.all
+  @tenants = Tenant.all
+  erb :"apartments/index"
+end
+
+get '/apartments/new' do
+  erb :"apartments/new"
+end
+
+get '/apartments/:id' do
+  @apartments = Apartment.all
+  @tenants = Tenant.all
+  @id = params[:id].to_i
```

Comment:
You have the right pieces, just not put together quite right.  This route is expecting to show a single apartment.  
```
@apartment = Apartment.find(params[:id]) # rails takes care of string/int conversion
```
Then, use @apartment in the show view.